src/ProxySettings.cpp

```diff
@@ -74,40 +66,54 @@ Napi::String ProxySettings::dump(const Napi::CallbackInfo& info)
        str.append("ProxyRead:Ok,");
        std::string::size_type portOffset = server.find_last_of(L':');
         if (portOffset != std::string::npos) {
-            str.append("server=").append(server.substr(0, portOffset));
+            str.append("host=").append(server.substr(0, portOffset));
             str.append(",port=").append(server.substr(portOffset + 1));
         } else {
-            str.append("server+port=").append(server).append(",delimiter not found!");
+            str.append("host+port=").append(server).append(",delimiter not found!");
         }
+    } else if (storage.get("AutoConfigURL", str) && !str.empty()) {
+        str.append("Proxy enabled: using pac script ").append(str);
```

Comment:
we're appending this message onto the url config value, it should be cleared first